src/Features/CSharp/Portable/CodeFixes/Suppression/CSharpSuppressionCodeFixProvider.cs

```diff
@@ -79,7 +79,7 @@ protected override bool IsAttributeListWithAssemblyAttributes(SyntaxNode node)
         }
 
         protected override bool IsEndOfLine(SyntaxTrivia trivia)
-            => trivia.Kind() == SyntaxKind.EndOfLineTrivia;
+            => trivia.IsKind(SyntaxKind.EndOfLineTrivia) || trivia.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia);
```

Comment:
Please make the same change in VB override and add unit tests for VB as well.

Reply:
➡️ I'll leave the VB side for a follow-up. I haven't hit any problems there myself, and haven't heard of any problems from users, so I'll leave it until such time as someone has a need for the change.